### PR TITLE
Info-table

### PR DESCRIPTION
**Pull request by OliverGreen27**
Black info-table in filereport moved to bottom right corner to reduce clutter in the code. It is responsive for different window sizes.

### CODE REVIEW

File: plugin/mutate/source/dextool/plugin/mutate/backend/report/html/page_mutant.d [hunk 1]
```diff
@@ -158,6 +153,7 @@ void makeAllMutantsPage(ref Database db, const(Mutation.Kind)[] kinds, const Abs
 
     long[MutantStatus] statusCnt;
     addMutants(db, kinds, tabContent, statusCnt);
+    // smurf smurf
```
Comment (joakim-brannstrom) [inline plugin/mutate/source/dextool/plugin/mutate/backend/report/html/page_mutant.d:156]: remove

File: plugin/mutate/data/source.js [hunk 2]
```diff
@@ -583,22 +596,32 @@ function init_legend() {
...
-function init_filter_status() {
-    var table = document.getElementById("filter_status");
 
-    for (var i=0; i<g_mut_st_map.length; i++) {
-        var row = table.insertRow(i);
-        row.innerHTML = `<td><input id="`+g_mut_st_map[i]+ `" type="checkbox" onclick='click_filter_status(`+i+`)' checked />
-        <span class="xx_label">`+g_mut_st_map[i]+`</span></td>`;
-    }
-    return;
+function hide_kind(){
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/source.js:610]: leftover code from experimenting?

File: plugin/mutate/data/source.js [hunk 3]
```diff
@@ -583,22 +596,32 @@ function init_legend() {
...
+  var table = document.getElementById("filter_kind");
+  if (table.style.display === "none") {
+    table.style.display = "block";
+  } else {
+    table.style.display = "none";
+  }
+}
+
+function hide_status(){
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/source.js:619]: leftover code from experimenting?

File: plugin/mutate/data/source.js [hunk 4]
```diff
@@ -224,7 +219,7 @@ function set_info_line(mutid) {
         }
         var test_case = tcs[i];
         if (test_case) {
-            row.innerHTML = make_td(test_case + ": "+g_testcases_kills[test_case]);
+            row.innerHTML = make_td(test_case + ": "+g_testcases_kills[test_case], "temp_id");
```
Comment (joakim-brannstrom) [inline plugin/mutate/data/source.js:222]: space on the sides of `+`

File: plugin/mutate/data/source.js (hunk 2 above)
Comment (OliverGreen27) [reply plugin/mutate/data/source.js:610]: We agreed on removing filter by kind and filter by status from the filereport tool. Should I remove all past functions that are were used for them but are not used anymore?

**Pull request merged**